.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: "Semantic PR"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      # Please look up the latest version from
+      # https://github.com/amannn/action-semantic-pull-request/releases
+      - uses: amannn/action-semantic-pull-request@v3.4.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed.
+          # Default: https://github.com/commitizen/conventional-commit-types
+          types: |
```

Comment:
Can we remove `types` to cause it to just use the default list?

Reply:
I personally like the availability to see at a glance what is expected. This workflow file should not get longer that that anyway...